lib/timers.js

```diff
@@ -377,7 +377,7 @@ function enroll(item, msecs) {
 exports.enroll = util.deprecate(enroll,
                                 'timers.unenroll() is deprecated. ' +
```

Comment:
s/unenroll/enroll/ ?

Reply:
The deprecation code is for `enroll` below, but the message is about `unenroll()`.

Reply:
@chalker done